common/conf/env/DEV.properties

```diff
@@ -93,3 +93,6 @@ standalone.oauth.callback=http://localhost:9000/oauth2callback
 
 # Beacon diagnostics
 beacon.url=//localhost:9000
+
+# Push Notifications
+push_notifications.host=http://localhost:9000
```

Comment:
Out of the box, this setup won't work. This property file assumes frontend *and* mobile-notifications are running on port 9000. Not sure what to do, because it's nigh impossible to change the port of mobile notifications due to hardcoded Google auth configuration.